Mathlib/Algebra/Quaternion.lean

```diff
@@ -572,6 +585,19 @@ theorem finrank_eq_four [StrongRankCondition R] : FiniteDimensional.finrank R 
   rw [FiniteDimensional.finrank, rank_eq_four, Cardinal.toNat_ofNat]
 #align quaternion_algebra.finrank_eq_four QuaternionAlgebra.finrank_eq_four
 
+/-- There is a natural equivalence when swapping the coefficients of a quaternion algebra. -/
+def swapEquiv : ℍ[R,c₁,c₂] ≃ₐ[R] ℍ[R, c₂, c₁] where
```

Comment:
In the context of `ℍ`, this doesn't seem natural at all; it treats `i` and `j` specially, without swapping the others!

I suppose this *is* natural for `ℍ[R,c₁,c₂]`, though I believe it is still available as dot notation on `Quaternion`.

Reply:
I mean, technically it turns `ij` into `ji = -ij` in that special case, I think. Did I mess up the namespacing?

Reply:
No, but `Quaternion := QuaternionAlgebra _ _ _` so dot notation naturally falls back on this definition.

Reply:
I didn't know dot notation could see through `def`s. Not sure what's best to do in this case. This is obviously not the standard involution...